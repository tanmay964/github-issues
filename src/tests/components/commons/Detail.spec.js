import React from 'react';
import renderer from 'react-test-renderer';
import Detail from '../../../components/commons/Detail';

it('renders correctly', () => {
  const tree = renderer
    .create(<Detail
      tagBackgroundColor="#fff"
      valueBackgroundColor="#fff"
      tagPadding="10px 10px"
      valuePadding="10px 10px"
      tag="Hey"
      value="test!"
      valueHref="http://github.com"
      tagHref="http://gitlab.com"
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
