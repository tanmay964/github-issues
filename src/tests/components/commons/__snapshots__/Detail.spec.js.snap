// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa ketRLB"
>
  <a
    className="sc-bwzfXH kajPdv"
    href="http://gitlab.com"
  >
    Hey
  </a>
  <a
    className="sc-htpNat iIYexs"
    href="http://github.com"
  >
    test!
  </a>
</div>
`;
